lib/dry/system/auto_registrar/configuration.rb

```diff
@@ -28,10 +28,13 @@ def self.setting(name)
         setting :exclude
         setting :instance
 
+        attr_accessor :memoize
+
```

Comment:
@solnic I don't remember a real reason why actually. We can use `dry-configurable`.

Would you like it to be done in this PR?

Reply:
> @GustavoCaso Would you like it to be done in this PR?

No. I was just curious because I can't remember either.

> @arielvalentin Does dry-configurable support the desired syntax? 

It does. We can consider using it instead but that's for another PR.